_posts/en/newsletters/2023-02-15-newsletter.md

```diff
@@ -0,0 +1,259 @@
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Core Lightning #5361][] adds experimental support for peer storage
+  backups.  As last mentioned in [Newsletter #147][news147 backups],
+  this allows a node to store a small encrypted backup file for its
+  peers.  If a peer later needs to reconnect, perhaps after losing data,
+  it can request the backup file.  The peer can use a key derived from
+  its wallet seed to decrypt the file and use the contents to recover
+  the latest state of all of its channels.  This can be seen as an
+  enhanced form of [static channel backups][topic static channel
+  backups].  This PR adds support for creating, storing, and retrieving
+  the encrypted backups.  As noted in this PR's commits, the feature
+  hasn't yet been fully specified or adopted by other LN
+  implementations.
+
+- [Core Lightning #5670][] and [#5956][core lightning #5956] makes
+  various updates to its implementation of [dual funding][topic dual
+  funding] based on both recent changes to the [specification][bolts
+  #851][] and comments from interoperability testers.  Additionally, an
+  `upgradewallet` RPC is added to move all funds in P2SH-wrapped outputs
+  to native segwit outputs, which is required for interactive channel
+  opens.
+
+- [Core Lightning #5697][] adds a `signinvoice` RPC that will sign a
+  [BOLT11][] invoice.  Previously, CLN would only sign an invoice when
+  it had the preimage for the [HTLC][topic HTLC] hash, ensuring it would
+  be able to claim a payment to the invoice.  This RPC can override
+  that behavior, which could (for example) be used to send an invoice
+  now and later use a plugin to retrieve the preimage from another
+  program.  Anyone using this RPC should be aware that any third-party
+  which has pre-knowledge of the preimage for a payment intended for
+  your node can claim that payment before it arrives.  That not only
+  steals your money but, because you signed the invoice, generates very
+  convincing evidence that you were paid (this evidence is so convincing
+  that many LN developers call *proof of payment*).
+
+- [Core Lightning #5960][] adds a [security policy][cln security.md]
+  that includes contact addresses and PGP keys.
+
+- [LND #7171][] upgrades the `signrpc` RPC <!--sic--> to support the
+  latest [draft BIP][musig draft bip] for [MuSig2].  The RPC now creates
+  sessions linked to a MuSig2 protocol version number so that all
+  operations within a session will use the correct protocol.  A
+  security issue with an older version of the MuSig2 protocol was
+  mentioned in [Newsletter #222][news222 musig2].
+
+- [LDK #2022][] adds support for automatically resending [spontaneous
+  payments][topic spontaneous payments] that don't initially succeed.
+
+- [BTCPay Server #4541][] makes a popular plugin for adding support for
```

Comment:
It is a bad sentence, and that's probably because I wasn't really sure what was happening.  I dropped it from this newsletter and will reconsider it for next week.  Thanks!